service/db/src/test/java/gov/va/vro/service/db/mapper/ClaimInfoResponseMapperTest.java

```diff
@@ -15,15 +15,22 @@
 import gov.va.vro.persistence.model.VeteranEntity;
 import gov.va.vro.service.db.TestConfig;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
 import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.transaction.annotation.Transactional;
 
 import java.util.List;
 import java.util.Map;
 
 @SpringBootTest(classes = TestConfig.class, webEnvironment = SpringBootTest.WebEnvironment.NONE)
+@Transactional
```

Comment:
This is just a mapper test; nothing JPA here. You do not need any of these new annotations.